themes/default/content/docs/intro/concepts/config.md

```diff
@@ -307,6 +307,72 @@ public static void stack(Context ctx) {
 
 {{< /chooser >}}
 
+## Project Level Configuration
+
+There are cases where configuration for more than one stack in a given project is the same. For example, `aws:region` may be the same across multiple or all stacks in a project. Project level configuration (also sometimes referred to as hieararchical configuration) allows setting configuration at the project level instead of having to repeat the configuration setting in each stack's configuration file.
+
+### Setting Project Level Configuration
+
+Project level configuration is defined inside the project folder's `Pulumi.yaml` file using one's favorite editor.
+
+{{% notes "info" %}}
+At this time, the `pulumi config set` command does not support project level configuration. Therefore the configuration values are entered directly in the `Pulumi.yaml` file. Also, project level configuration only supports clear text configuration. Support for secrets and other features are planned.
```

Comment:
Link to the issues?
https://github.com/pulumi/pulumi/issues/12041
https://github.com/pulumi/pulumi/issues/11549

Reply:
I hemmed and hawed about linking the issues. But, your vote broke the tie and I added them.